src/lib/features/client-feature-toggles/tests/client-feature-toggles.e2e.test.ts

```diff
@@ -135,7 +137,13 @@ const setupFeatures = async (
         {
             name: 'default',
             constraints: [
-                { contextName: 'userId', operator: 'IN', values: ['123'] },
+                {
+                    contextName: 'userId',
+                    operator: 'IN',
+                    values: ['123'],
+                    caseInsensitive: false,
+                    inverted: false,
```

Comment:
These two are needed in the input because we're using the method directly. If using the http API, openapi will set these undefined values to their respective default value.